charts/civil-service/values.preview.template.yaml

```diff
@@ -128,6 +128,7 @@ ccd:
         DATA_STORE_IDAM_KEY: ${CCD_DATA_STORE_S2S_SECRET}
         DATA_STORE_S2S_AUTHORISED_SERVICES: ccd_data,ccd_gw,ccd_ps,bulk_scan_orchestrator,ccpay_bubble,ctsc_work_allocation,em_ccd_orchestrator,xui_webapp,civil_service
         IDAM_USER_URL: https://idam-web-public.aat.platform.hmcts.net
+        IDAM_OIDC_URL:  https://idam-web-public.aat.platform.hmcts.net
```

Comment:
what's this change for @dharmendrak? Seem unrelated to bintray

Reply:
ccd have changed this config